### ISSUE TITLE
Click Accuracy!

### ISSUE THREAD

**Issue by OrrnobMahmud** [phen0menon/blum-autoclicker#2]
Hi, I am glad I finally found a bot that works, but I am sorry to say it's far from accuracy! I miss 70% of the clicks! So far it farms 70-80 point each session! Can you tell me if there is anything I have to do with mouse settings?

**Comment by phen0menon** [comment]
Hey @OrrnobMahmud! Note that the bot will work swiftly only with Pytorch with CUDA enabled. To enable this, you need to install additional package:

```
# uninstall existing packages first
pip uninstall torch torchvision torchaudio

pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
```

This way all PyTorch models will load via your GPU.

**Comment by phen0menon** [comment]
On my RTX 3070 the bot farms 300-350 points per game.

**Comment by OrrnobMahmud** [comment]
> On my RTX 3070 the bot farms 300-350 points per game.

Then that's might the issue, I only got an AMD Ryzen 5 5600G. I have installed all the requirements including 
`pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118` this one. It's my potato pc, then! :(

**Comment by phen0menon** [comment]
I believe that's the issue. If you don't have a GPU, the detection will work on your CPU which is way mooore slower than GPU for tasks like this.

**Comment by phen0menon** [comment]
I can try implementing a solution with CPU recognition only, without YOLO pretrained models, just by using CV. But it depends on how many users would like to see that feature.

**Comment by OrrnobMahmud** [comment]
I don't know how many users looking for it, But after week of searching. I found this one and the rest you know. BUT anyway, thanks for so much hard work you have already done. I have suggested this to many of my friends!
